### ISSUE TITLE
Create a standard for pull requests and commit messages

### ISSUE THREAD

**Issue by ClementineAccount** [ClementineAccount/Albuquerque#50]
**Summary**

Due to inexperience, commit messages and pull requests in the past for all my past projects, including Alberquerque, did not seek to meet any consistent standard. After today, I have decided to encourage better habits and a stride towards a more professional and pragmatic standard for version control messages. This will allow an easier transition into work environments and confidence in my abilities to comply with standards when joining the workforce upon graduation.

**Why?**

This pull request simply represents the transition period between attempts to uphold a standard for contributors of the project, whinc includes only myself for the foreseeable future, and the past 'wild west' of previous commits. No attempt to rebase the previous history is necessary, and in fact the preservation of previous commit messages and pull requests can serve as a valuable study and personal growth.

In short, this pull request would appear in a commit log as a 'divider' between the old and the new for anyone reading the history.
